src/run_solver.jl

```diff
@@ -46,13 +49,16 @@ function solve_problems(
     :dual_feas,
     :primal_feas,
   ],
-  info_hdr_override::Dict{Symbol, String} = Dict{Symbol, String}(:solver_name => "Solver"),
+  info_hdr_override::Dict{Symbol, String} = Dict(:solver_name => "Solver"),
   prune::Bool = true,
   kwargs...,
 ) where {TName}
+
+  # Collect information about counters
```

Comment:
Please remove these comments and blank lines that do not have anything to do with the new functionality.